mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/dao/DaoImplementationGenerator.java

```diff
@@ -298,7 +299,7 @@ protected ClassName getPrincipalTypeName() {
         TypeSpec.classBuilder(implementationName)
             .addJavadoc(JAVADOC_GENERATED_WARNING)
             .addModifiers(Modifier.PUBLIC)
-            .superclass(DaoBase.class)
+            .superclass(getDaoParentClass())
```

Comment:
This might be useful to do in `EntityHelperGenerator.getContents()` as well as we follow a similar pattern there with using an `EntityHelperBase` for `EntityHelper` implementations.

Reply:
Good idea, will do.